import.lua

```diff
@@ -5,22 +5,51 @@
 
     modules { -- Can be `module 'module_name'` as well
         'module_name', -- This is the name of the file in the modules folder in qbx-core without the .lua
-        'client:module_name' -- You can specify the side it should be loaded on like this
+        'client:module_name', -- You can specify the side it should be loaded on like this
+        'resourceName:module_name', -- You can also specify the resource it should be loaded from if you don't want to load it from qbx-core
+        'resourceName:client:module_name' -- The resource loading also accepts a side to load on
     }
 ]]
 
-local resourceName = 'qbx-core'
+local resources = {}
 local isServer = IsDuplicityVersion()
 
+local function getSide(value)
+    return value == 'client' and 'client' or value == 'server' and 'server' or value == 'shared' and 'shared' or 'none'
```

Comment:
```suggestion
    if value == 'client' or value == 'server' or value == 'shared' then return value
    else return 'none' end
```

nit; I find this way to be a bit more readable.

Reply:
It seems readable enough for me as it is, pretty straight forward